modules/services/dropbox.nix

```diff
@@ -33,48 +33,42 @@ in {
         lib.platforms.linux)
     ];
 
-    home.packages = [ pkgs.dropbox-cli ];
+    home.packages = [ pkgs.dropbox ];
 
     systemd.user.services.dropbox = {
       Unit = { Description = "dropbox"; };
 
       Install = { WantedBy = [ "default.target" ]; };
 
       Service = {
-        Environment = [ "HOME=${homeBaseDir}" "DISPLAY=" ];
+        Environment = [ "HOME=${homeBaseDir}" "LD_LIBRARY_PATH=${pkgs.libGL}/lib/"];
```

Comment:
This should be fixed in the package

Reply:
How do I do that? Sorry this is my first contribution.

Reply:
@ttuegel Can you help me with this?

Reply:
> This should be fixed in the package
@SuperSandro2000 

This has been fixed. Is this ready to be merged?